common/djangoapps/student/models/course_enrollment.py

```diff
@@ -596,7 +596,7 @@ def emit_event(self, event_name, enterprise_uuid=None):
                                                               self.course_overview.display_name,
                                                               self.course_overview.short_description,
                                                               self.course_overview.has_ended(),
-                                                              course_pacing_type))
+                                                              course_pacing_type, self.mode))
```

Comment:
nit: keep the code consistent and move the param to next line.
```suggestion
                                                              course_pacing_type, 
                                                              self.mode))
```